fog/view/server/src/fog_view_service.rs

```diff
@@ -119,6 +119,36 @@ where
         }
     }
 
+    fn auth_nonce_impl(
+        &mut self,
+        mut request: attest::AuthMessage,
+        logger: &Logger,
+    ) -> Result<attest::AuthMessage, RpcStatus> {
+        // TODO: Use the prost message directly, once available
+        match self.enclave.frontend_accept(request.take_data().into()) {
+            Ok((response, _)) => {
+                let mut result = attest::AuthMessage::new();
+                result.set_data(response.into());
+                Ok(result)
+            }
+            Err(client_error) => {
+                // This is debug because there's no requirement on the remote party to trigger
+                // it.
+                log::debug!(
+                    logger,
+                    "ViewEnclaveApi::frontend_accept failed: {}",
+                    client_error
+                );
+                let rpc_permissions_error = rpc_permissions_error(
+                    "client_auth",
+                    format!("Permission denied: {}", client_error),
```

Comment:
:wrench: 
```suggestion
            Err(frontend_error) => {
                // This is debug because there's no requirement on the remote party to trigger
                // it.
                log::debug!(
                    logger,
                    "ViewEnclaveApi::frontend_accept failed: {}",
                    frontend_error
                );
                let rpc_permissions_error = rpc_permissions_error(
                    "frontend_accept",
                    format!("Permission denied: {}", frontend_error),
```

Reply:
Done.